client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -58,6 +60,13 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () =>
 				<Route exact path={ Step.DesignSelection }>
 					{ ! siteVertical ? <Redirect to={ Step.IntentGathering } /> : <DesignSelector /> }
 				</Route>
+				<Route exact path={ Step.PageSelection }>
+					{ ! selectedDesign ? (
+						<Redirect to={ Step.DesignSelection } />
+					) : (
+						<DesignSelector showPageSelector={ true } />
```

Comment:
We typically use this JSX shorthand notation for `true` boolean props:

```suggestion
						<DesignSelector showPageSelector />
```

Reply:
Merged before I saw this. Will do that next time.

Reply:
Oh, funny that there isn't Eslint rule turned on for that.

Reply:
Indeed :thinking: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md